src/app/home/home.component.spec.ts

```diff
@@ -1,33 +1,25 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { HomeComponent } from './home.component';
-import { MatDialog, MatDialogModule } from "@angular/material/dialog";
-import { RoomCreationModalComponent } from "./room-creation-modal/room-creation-modal.component";
-import { of } from "rxjs";
+import { MatDialog } from '@angular/material/dialog';
+import { RoomCreationModalComponent } from './room-creation-modal/room-creation-modal.component';
+import { of } from 'rxjs';
 
 describe('HomeComponent', () => {
   let component: HomeComponent;
-  let dialogSpy: jasmine.Spy;
-  let dialogRefSpyObj = jasmine.createSpyObj({ afterClosed : of({}), close: null });
-  dialogRefSpyObj.componentInstance = { body: '' }
-  let fixture: ComponentFixture<HomeComponent>;
 
-  beforeEach(async () => {
-    await TestBed.configureTestingModule({
-      declarations: [ HomeComponent ],
-      imports: [ MatDialogModule ]
-    })
-    .compileComponents();
+  let dialogSpy: jasmine.Spy;
+  let dialogRefSpyObj = jasmine.createSpyObj({ afterClosed: of({}), close: null });
+  dialogRefSpyObj.componentInstance = { body: '' };
 
-    fixture = TestBed.createComponent(HomeComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
+  beforeEach(() => {
+    const dialog = { open: (_1, _2) => {} } as MatDialog;
+    dialogSpy = spyOn(dialog, 'open').and.returnValue(dialogRefSpyObj);
 
-    dialogSpy = spyOn(TestBed.get(MatDialog), 'open').and.returnValue(dialogRefSpyObj);
+    component = new HomeComponent(dialog);
   });
 
   it('opens the dialog with configuration', () => {
     component.openDialog();
 
-    expect(dialogSpy).toHaveBeenCalledWith(RoomCreationModalComponent, { width: "20rem" });
```

Comment:
Maybe a lint rule?
![image](https://user-images.githubusercontent.com/17585880/195842118-21b96d89-5ed0-48c9-aa2c-1378c7b27e23.png)

Reply:
I don't get it, what is the problem here?

Reply:
Maybe you are

Reply:
Add a rule that makes the use of `"` fails the lint, so we will be forced to use `'`

Reply:
Ignore as discussed :) It is already configured correctly, just CI doesn't run lint it seems.